tests/base/async.js

```diff
@@ -47,7 +47,6 @@ if (isMainThread) {
         console.log('train task is completed');
         setTimeout(() => {
           clearInterval(alive);
-          console.log(foo.ping('x'));
```

Comment:
[F] This call is to an object that is in the middle of garbage collection. I could get the call to work by moving it outside of `setTimeout` on node v14 but not v12. I would assume that is because garbage collection works slightly differently between the two versions. I would also assume that the test worked previously because the object was not being collected at this point. Please let me know if you disagree with these assumptions. 

Thanks